doc/dev.rst

```diff
@@ -1246,3 +1246,23 @@ Additional sources
 * https://github.com/opencontainers/runtime-spec/blob/master/runtime.md
 
 ..  LocalWords:  milestoned gh nv cht Chacon's scottchacon
+
+Updating bundled Lark Parser
+============================
+
+In order to change the version of the blunded lark parser you must first find
+the commit you would like to change the lark version to. Go to
+https://github.com/lark-parser/lark/tags and choose a tagged release.
```

Comment:
This is a little unclear. It says I need a specific commit then says to choose a tagged release. I suspect it can be either; consider revising.

Reply:
Tagged releases are specific commits. But I'll see if I can be more clear without too much text.